docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,559 @@
+
+Terraform can be opaque in the beginning. If you solely want to get an understanding for what is happening, you may try out the [eksctl guide](./eksctl.md) to understand what resources are created and how they interact with each other.
+
+To try out Camunda 8 or develop against it, consider signing up for our [SaaS offering](https://camunda.com/platform/). If you already have two Amazon EKS clusters, which are peered together and a S3 bucket, consider skipping to [Deploy Camunda 8 to the Clusters](#deploy-camunda-8-to-the-clusters).
+
+For the simplicity of this guide, certain best practices will be provided with links to additional documents, enabling you to explore the topic in more detail.
+
+:::warning
+Following this guide will incur costs on your Cloud provider account, namely for the managed Kubernetes service, running Kubernetes nodes in EC2, Elastic Block Storage (EBS), traffic between regions, and S3. More information can be found on [AWS](https://aws.amazon.com/eks/pricing/) and their [pricing calculator](https://calculator.aws/#/) as the total cost varies per region.
+:::
+
+:::warning
+Please have a look at our [dual-region concept documentation](#) <!-- TOOD: link reference --> before continuing to understand the current limitations and restrictions of this setup, as well as the disclaimer concerning support from Camunda.
+:::
+
+## Outcome
+
+Following this tutorial and steps will result in:
+
+- Two Amazon EKS Kubernetes clusters running the latest Kubernetes version with each four nodes ready for Camunda 8 installation.
+- The [EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) is installed and configured, which is used by the Camunda 8 Helm chart to create [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+- A [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) between the two EKS clusters, allowing cross-cluster communication between different regions.
+- An [Amazon Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) (S3) bucket for [ElasticSearch backups](https://www.elastic.co/guide/en/elasticsearch/reference/current/repository-s3.html)
+
+## Environment prerequisites
+
+To streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values:
+
+```bash
+# The AWS regions of your cluster 1 and 2
+export REGION_1=eu-west-2
+export REGION_2=eu-west-3
+
+# The cluster_name of your cluster 1 and 2
+# default based on the tutorial is the following
+export CLUSTER_1=cluster-london
+export CLUSTER_2=cluster-paris
+
+# The namespaces for each region where Camunda 8 should be running in
+export CAMUNDA_NAMESPACE_1=camunda-london
+export CAMUNDA_NAMESPACE_2=camunda-paris
+```
+
+## Installing Amazon EKS clusters with Terraform
```

Comment:
## **[big one]**

I have a general feeling it's too much work for us to guide the user what files to create and what content to put there.

Maybe we can say the user needs to clone https://github.com/camunda/c8-multi-region/tree/aws-operational/test/resources/aws/2-region/terraform (with all files pre-created) and customize the Terraform variables and then just `tf apply`?

It's cool we can embed the code content, but I also have a feeling it won't be much useful for the user and create a lot of work for us to maintain/create such content.

wdyt?

Reply:
the proposed change is included in the latest commit - https://github.com/camunda/camunda-docs/pull/3430/commits/4e11aa0a397cdf49f8810c07b254f8420fb23e58